implementations/rust/ockam/ockam_api/src/enroll/ockam_oidc_provider.rs

```diff
@@ -1,11 +1,19 @@
+use ockam_core::env::get_env;
 use ockam_core::Result;
 use std::time::Duration;
 use url::Url;
 
 use crate::enroll::oidc_provider::OidcProvider;
 
+pub fn authenticator_endpoint() -> String {
+    get_env("OCKAM_AUTHENTICATOR_ENDPOINT")
```

Comment:
Not sure if it's relavent, but can this be added to this page as well similar to OCKAM_CONTROLLER_ADDR and OCKAM_CONTROLLER_IDENTITY_ID  - Those are the two values we set explicitly during development environment testing. 

And i'm assuming the default value for OCKAM_AUTHENTICATOR_ENDPOINT will be set to https://account.ockam.io rite? Asking because I didn't see that set any place.

Reply:
Sure! I wasn't sure about doing it because it's on a separate crate... but it's also the only file where we keep track of the env vars.

Reply:
> And i'm assuming the default value for OCKAM_AUTHENTICATOR_ENDPOINT will be set to https://account.ockam.io/ rite? Asking because I didn't see that set any place.

I thought `https://account.ockam.io/` was going to be replaced by something else. Is that still the default value?

Reply:
Yes `https://account.ockam.io/` is the default value to compile the binaries with. During testing, I shall export `OCKAM_AUTHENTICATOR_ENDPOINT` to another value to match our development environment. Similar `OCKAM_CONTROLLER*` values we override during development environment testing